scripts/L0Config.json

```diff
@@ -9,7 +9,14 @@
             "eid": 30101,
             "endpoint": "0x1a44076050125825900e736c501f859c50fE728c",
             "receiveLib302": "0xc02Ab410f0734EFa3F14628780e6e695156024C2",
-            "sendLib302": "0xbB2Ea70C9E858123480642Cf96acbcCE1372dCe1"
+            "sendLib302": "0xbB2Ea70C9E858123480642Cf96acbcCE1372dCe1",
+            "contractName":"",
+            "frxUSD":"",
```

Comment:
Note that from [here](https://github.com/FraxFinance/frax-oft-upgradeable/blob/master/scripts/BaseL0Script.sol#L42) the struct needs to be defined in alphabetical order.